src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -263,8 +260,7 @@ package_1
 ## Publishing packages {#publish}
 
 Once you have implemented a package, you can publish it on the
-[Pub site]({{site.pub}}), so that other developers can
-easily use it.
+[pub.dev][] site, so that other developers can easily use it.
```

Comment:
GLOBAL: "on the [pub.dev][] site -> "on the [pub.dev site][]" or "on [pub.dev][]"

(i.e. if you use "site", put it in the link)